### PR TITLE
add `extend_prev_word_end` command

### PR DESCRIPTION
**Pull request by asvln**
Adds `extend_prev_word_end` to user callable commands.

### CODE REVIEW

File: helix-term/src/commands.rs [hunk 1]
```diff
@@ -310,8 +311,7 @@ impl MappableCommand {
         goto_line_end, "Goto line end",
         goto_next_buffer, "Goto next buffer",
         goto_previous_buffer, "Goto previous buffer",
-        // TODO: different description ?
-        goto_line_end_newline, "Goto line end",
+        goto_line_end_newline, "Goto line end (newline)",
         goto_first_nonwhitespace, "Goto first non-blank in line",
```
Comment (the-mikedavis) [inline helix-term/src/commands.rs:315]: Looks like an unrelated change?

Comment (the-mikedavis) [reply helix-term/src/commands.rs:315]: Might as well rename this though. I would prefer "Goto newline at line end"

Comment (asvln) [reply helix-term/src/commands.rs]: Apologies, I forgot I did that when reordering. I have updated to description.

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thank you!
